src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -0,0 +1,243 @@
+                }
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void Replace(int size)
+        {
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.Replace("-", "1");
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void Split(int size)
+        {
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.Split("-");
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void StartsWith(int size)
+        {
+            string testString = PerfUtils.CreateString(size);
+            string subString = testString.Substring(0, testString.Length / 4);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.StartsWith(subString);
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void Empty(int size)
+        {
+            string result;
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                {
+                    result = string.Empty; result = string.Empty; result = string.Empty;
+                    result = string.Empty; result = string.Empty; result = string.Empty;
+                    result = string.Empty; result = string.Empty; result = string.Empty;
+                }
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void Substring_int(int size)
+        {
+            int startIndex = size / 2;
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.Substring(startIndex);
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void Substring_int_int(int size)
+        {
+            int startIndex = size / 2;
+            int length = size / 4;
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.Substring(startIndex, length);
+        }
+
+        [Benchmark]
+        [MemberData("TestStringSizes")]
+        public void ToLower(int size)
+        {
+            string testString = PerfUtils.CreateString(size);
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    testString.ToLower();
```

Comment:
Guid.NewGuid().ToString() produces a string with numbers and all lowercase letters... this may not be testing what you want to be testing.